operations/loyaltyprograms.md

```diff
@@ -85,4 +85,153 @@ Note this operation uses [Pagination](../guidelines/pagination.md).
 | :-- | :-- | :-- | :-- |
 | `Id` | string | required | Unique identifier of the loyalty program. |
 | `Name` | string | required | Name of the loyalty program. |
-| `Code` | string | required | Code of the loyalty program. |
\ No newline at end of file
+| `Code` | string | required | Code of the loyalty program. |
+
+## Add loyalty programs
+
+Adds loyalty programs to the enterprise (in the given activity state).
```

Comment:
The activity state isn't part of the operation at all.

Reply:
Copy paste error :) fixing 👍🏻